test/IRGen/objc_simd.sil

```diff
@@ -1,4 +1,4 @@
-// RUN: %target-swift-frontend(mock-sdk: %clang-importer-sdk) -emit-ir %s | %FileCheck %s --check-prefix=%target-cpu
+// RUN: %target-swift-frontend(mock-sdk: %clang-importer-sdk) -emit-ir %s | %FileCheck %s --check-prefix=%target-cpu --check-prefix=%target-sdk-name
```

Comment:
From the check lines, it looks like you want this to be `--check-prefix=%target-cpu-%target-sdk-name`.

Reply:
Arggs. I'm sorry. It is indeed. I hate that lit doesn’t warn or anything when nothing matches a line.